SPECS/kernel/kernel.spec

```diff
@@ -3,7 +3,7 @@
 Summary:        Linux Kernel
 Name:           kernel
 Version:        5.4.83
-Release:        1%{?dist}
+Release:        2%{?dist}
```

Comment:
Please bump the release numbers of the kernel-signed-<arch> specs

Reply:
good catch! I have updated all related kernel files now.